history-network.md

```diff
@@ -112,20 +102,8 @@ def update_accumulator(accumulator: MasterAccumulator, new_block_header: BlockHe
     accumulator.current_epoch.append(header_record)
 ```
 
-#### Growth over time
-
-Each `HeaderRecord` is 64 bytes meaning the `EpochAccumulator` can range from `0 - 64 * EPOCH_SIZE` bytes (0-512KB) over the course of a single epoch.
-
-The `Accumulator.historical_epochs` grows by 32 bytes per epoch.
-
-At a 13 second block time we expect:
+The network provides no mechanism for acquiring the *master* version of this accumulator.  Clients are encouraged to solve this however they choose, with the suggestion that they include a frozen copy of the accumulator at the point of the merge within their client code, and provide a mechanism for users to override this value if they so choose.
```

Comment:
I think that is desired.  The network is unlikely to reliably host "minority" accumulators and the master accumulator would be static, thus it would live in a static location, and thus the nodes in that location would be subject to DOS issues if lots of people requested it from them....